src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -26,47 +26,47 @@ private Messages() {
         throw new RuntimeException("Could not construct class");
     }
     //Universal Messages
-    public static Component setUniversalPlayerOnlyCommandErrorMessage() {
+    public static Component getUniversalPlayerOnlyCommandError() {
```

Comment:
Remove ``Universal``

Reply:
oh yes ok XD

Reply:
just me bein dumb

Reply:
It should be assumed that all messages are universal. Adding the word univeral to only some may confuse newcomers

Reply:
oh damn wait